Error running npx @contentful/create-contentful-app init

Hi folks,

I'm trying to create a new Contentful app, using this package, but I keep receiving the following error:

```
➟ npx @contentful/create-contentful-app init my-app
npx: installed 140 in 5.031s
Creating a Contentful app in ~/Projects/my-app.

Creating a new React app in ~/Projects/my-app.

Installing packages. This might take a couple of minutes.
Installing react, react-dom, and react-scripts with @contentful/cra-template-create-contentful-app...

npm ERR! code ETARGET
npm ERR! notarget No matching version found for @contentful/cra-template-create-contentful-app@0.8.1.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     ~/.npm/_logs/2021-04-21T18_02_59_303Z-debug.log
```

Seems like it's trying to install `@contentful/cra-template-create-contentful-app@0.8.1` which is not published yet.

NPM tells me the [latest available version](https://www.npmjs.com/package/@contentful/cra-template-create-contentful-app) is 0.8.1.alpha.2

--- a/package.json
+++ b/package.json
@@ -12,8 +12,8 @@
     "test": "lerna run test",
     "lint": "lerna run lint",
     "link": "lerna link",
-    "version": "lerna version --no-private --conventional-commits --create-release github --yes --exact --force-publsih",
-    "publish": "lerna publish from-git --yes --no-verify-access --exact --force-publsih",
+    "version": "lerna version --no-private --conventional-commits --create-release github --yes --exact --force-publish",
+    "publish": "lerna publish from-git --yes --no-verify-access --exact --force-publish",
     "pre-commit": "lerna run pre-commit",
     "commit-msg": "lerna run commit-msg"
   },

--- a/package.json
+++ b/package.json
@@ -12,8 +12,8 @@
     "test": "lerna run test",
     "lint": "lerna run lint",
     "link": "lerna link",
-    "version": "lerna version --no-private --conventional-commits --create-release github --yes --exact",
-    "publish": "lerna publish from-git --yes --no-verify-access --exact",
+    "version": "lerna version --no-private --conventional-commits --create-release github --yes --exact --force-publsih",
+    "publish": "lerna publish from-git --yes --no-verify-access --exact --force-publsih",
     "pre-commit": "lerna run pre-commit",
     "commit-msg": "lerna run commit-msg"
   },

